.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       uses: codecov/codecov-action@v3
     - name: Extract branch name
       shell: bash
-      run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      run: echo "branch=$(echo ${GITHUB_REF#refs/heads/})" >> $GITHUB_ENV
```

Comment:
I don't think this will work, In the [github examples](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/#examples) it says that you should replace `set-output` to writing to`$GITHUB_OUTPUT` env file, not `$GITHUB_ENV`

If you write to `$GITHUB_ENV` the result will not be available in our last ci step in this file where we do `run: npx semantic-release --dry-run --branches=${{ steps.extract_branch.outputs.branch}}`, right now `steps.extract_branch.outputs.branch` returns empty string, you can see it in the [logs](https://github.com/openedx/paragon/actions/runs/6896611271/job/18763070440) (in the `Preview semantic-release version`)

Reply:
Thank you! When testing I forgot to change the file name